demo/wp-feature-api-demo/wp-feature-api-demo.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Plugin Name: WordPress Feature API Demo
+ * Plugin URI: https://github.com/Automattic/wp-feature-api
+ * Description: Demo plugin showcasing the WordPress Features API.
+ * Version: 0.1.0
+ * Author: WordPress Contributors
+ * Author URI: https://wordpress.org/
+ * Text Domain: wp-feature-api-demo
+ * License: GPL-2.0+
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
+ * Requires at least: 6.0
+ * Requires PHP: 7.2
+ *
+ * @package WordPress\Feature_API_Demo
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'wp_register_feature' ) ) {
```

Comment:
This check needs to be done after the plugins are loaded, otherwise, the check may fail simply because the code of the other plugin was not yet loaded. On my setup, this check is failing.